pkg/registry/batch/cronjob/strategy_test.go

```diff
@@ -428,3 +433,161 @@ func TestJobStrategy_WarningsOnUpdate(t *testing.T) {
 		})
 	}
 }
+
+var (
```

Comment:
Move the `cronJobPodFailurePolicyIn0` var declaration to the test where it is used. If more tests start to use this variable over time it will be hard to say if we can modify it without affecting the intention.